models/glif_psc_double_alpha.h

```diff
@@ -0,0 +1,481 @@
+========= ======== ============================================================
+
+========================= =============== =====================================
+**Spike adaptation and firing intensity parameters**
+-------------------------------------------------------------------------------
+th_spike_add               double         Threshold addition following spike
+                                          in mV (delta_theta_s in Equation (6)
+                                          in [1]_)
+th_spike_decay             double         Spike-induced threshold time
+                                          constant in 1/ms (bs in Equation (2)
+                                          in [1]_)
+voltage_reset_fraction     double         Voltage fraction coefficient
+                                          following spike (fv in Equation (5)
+                                          in [1]_)
+voltage_reset_add          double         Voltage addition following spike in
+                                          mV (-delta_V (sign flipped) in
+                                          Equation (5) in [1]_)
+asc_init                   double vector  Initial values of after-spike
+                                          currents in pA
+asc_decay                  double vector  After-spike current time constants
+                                          in 1/ms (kj in Equation (3) in [1]_)
+asc_amps                   double vector  After-spike current amplitudes in
+                                          pA (deltaIj in Equation (7) in [1]_)
+asc_r                      double vector  Current fraction following spike
+                                          coefficients for fj in Equation (7)
+                                          in [1]_
+th_voltage_index           double         Adaptation index of threshold - A
+                                          'leak-conductance' for the
+                                          voltage-dependent component of the
+                                          threshold in 1/ms (av in Equation
+                                          (4) in [1]_)
+th_voltage_decay           double         Voltage-induced threshold time
+                                          constant - Inverse of which is the
+                                          time constant of the
+                                          voltage-dependent component of the
+                                          threshold in 1/ms (bv in Equation
+                                          (4) in [1]_)
+tau_syn_fast               double vector  Rise time constants of the faster
+                                          synaptic alpha function in ms
+tau_syn_slow               double vector  Rise time constants of the slower
+                                          synaptic alpha function in ms
+amp_slow                   double vector  Relative amplitude of the slower
```

Comment:
Sorry, but I find it a bit hard to interpret the meaning of the parameters. Could you perhaps express the double-alpha function mathematically?

Something like, "there are as many alpha functions as ``len(amp_slow)``; then, I_syn_total = \sum_i alpha_function[tau_rise = tau_rise_slow[i], tau_decay = tau_decay_slow[i]] + alpha_function[tau_rise = tau_rise_fast[i], tau_decay = tau_decay_fast[i]]". Thanks!

Reply:
Please see my comment above. I put down an equation in the Description in the file and changed the 'Rise time constants' to 'Time constants' because, for alpha functions, there is only one time constant. I made the same change to the original glif_psc.h as well.

Reply:
Great, thank you!